src/DB_Command.php

```diff
@@ -258,6 +261,8 @@ public function check( $_, $assoc_args ) {
 
 		WP_CLI::debug( 'Associative arguments: ' . json_encode( $assoc_args ), 'db' );
 		WP_CLI::success( 'Database checked.' );
+		WP_CLI::log( 'To check whether the WordPress DB tables are installed: wp core is-installed', 'db' );
+		WP_CLI::log( 'To check whether the WordPress DB tables need to be updated: wp core update-db --dry-run', 'db' );
```

Comment:
We generally don't render any output after `WP_CLI::success()`.

For this particular problem, I'd prefer to remove these `WP_CLI::log()` statements and simply keep to the command doc changes.

Reply:
Removed in https://github.com/wp-cli/db-command/pull/244/commits/1debf788f416b059d085b47ed5f4628c538495cf